test/support/helper.js

```diff
@@ -32,7 +32,10 @@ global.san = san
 global.sinon = sinon
 global.sandbox = sandbox
 
-beforeEach(() => san.PropertiesService.init())
+beforeEach(() => {
+  san.PropertiesService.init()
+  san.CacheService.init()
```

Comment:
Why the cache service is initialized before every test?

Reply:
Isn't that initialized before each test session? Asking mainly, because when there is a result in one of the tests, if another test has the same result, it fetches the cached result